backend/inspirehep/records/marshmallow/journals/ui.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from marshmallow import fields
+
+from inspirehep.records.marshmallow.journals.base import JournalsPublicListSchema
+
+class JournalsBaseSchema(JournalsPublicListSchema):
+    pass
+
+class JournalsListSchema(JournalsBaseSchema):
+    public_notes = fields.Raw()
+    short_title = fields.String()
+    publisher = fields.Raw()
+    journal_title = fields.Method("get_journal_title", dump_only=True)
+    urls = fields.Raw()
+
+    def get_journal_title(self, original_object):
+        """Populate the ``journal_title`` field of Journals records."""
+        journal_title = original_object.get_value("journal_title.title", default="")
```

Comment:
to me it would be better to have it in a base schema since its title is kind of more generic